### DISCUSSION TITLE
Can't log in using Laravel Breeze and Sanctum

### DISCUSSION THREAD

**Discussion by oladejihenry** [jenssegers/laravel-mongodb#3023] [Q&A]
I am having trouble signing user in to my application using Nextjs and laravel breeze (sanctum api) anytime i try to log user in i get this error.

```
"message": "This database engine does not support inserting while ignoring errors.",
    "exception": "RuntimeException",
    "file": "/var/www/html/vendor/laravel/framework/src/Illuminate/Database/Query/Grammars/Grammar.php",
    "line": 1179,
    
    ```
    
    Also i wasn't able to register user until i changed SESSION_DRIVER from database to file
    
    Any help will be appreciated
    
    I am using the default laravel breeze AuthenticatedSessionController for the signing in

**Comment by bisht2050** [comment]
[PHPORM-88](https://jira.mongodb.org/browse/PHPORM-88) tracks the official support for Sanctum.

**Reply by oladejihenry** [reply]
I think what i am going to do for now is have session and user on another database like Postgres and have other Articles on Mongodb. That should solve my problem till sanctum is fully supported.

**Reply by masterbater** [reply]
> I think what i am going to do for now is have session and user on another database like Postgres and have other Articles on Mongodb. That should solve my problem till sanctum is fully supported.

It should be supported in 4.6
